### DISCUSSION TITLE
Ekip Kurma Fikirleri neler tartışalım.

### DISCUSSION THREAD

**Discussion by sercangoger** [acikkaynak/deprem-yardim-projesi#59] [Q&A]
arkadaşla herkes fikirlerini sunuyor. fakat ekip kurma konusunda bu projelere insanların atanması konusunda bir fikir biirliğinin oluşturulması projelere başlanılması daha iyi oalcaktır. bu yüzden herkes bu takımların nasıl kurulması nasıl lead edilmesi konusunda bilgilerin paylaşarak bunun için roller yapılabilir projelere başlanabilir.

discord da deprem.io içerisinde yapılacak proje için kanalın oluşturulması ve direk gönüllülerin kanala kaydedip kendilierini tanıtmalı.

her proje için

bir takım lideri
en az 2 Frontend
en az 2 Backend
2 UI Design
en az 1 Cloudcu

gibi kişileri kanalda toplayıp takım liderini koordine edecek arkadşın takım liderleri ile görüşüp gerekli repo ve diğer desteklerin sağlanıp koordinasyonumuzu sağlaması önemli olacaktır.

lütfen sizde fikirlerinizi giriniz ve bunun üzerine hızlı aksiyon alabiliriz

**Comment by ramazansancar** [comment]
Merhaba arkadaşlar,
Öncelikle fikriniz için teşekkürler. Şu an discord düzeni bu şekildedir. Sadece ekipte görevli kişiler kanallara erişebilmektedir.

Bkz. 
![image](https://user-images.githubusercontent.com/8878055/218271313-a5a8b5c2-016d-44f8-b322-7b3889a2467b.png)
